.github/workflows/test.yaml

```diff
@@ -105,3 +113,33 @@ jobs:
       run: |
         python --version
         tox -q
+
+    - name: upload dist
+      uses: actions/upload-artifact@v2
+      with:
+        name: py${{ matrix.python-version }}_dist
+        path: dist
+
+
+  all-successful:
```

Comment:
can we have a more descriptive name for this.
Maybe 'pypi_publish' ?

Reply:
```suggestion
  pypi-publish:
```